examples/text_classification/tc_mnli.ipynb

```diff
@@ -0,0 +1,6301 @@
+{
```

Comment:
<p>Unify the cases in the title? Some words start with capital while some start with lower case right now. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp/pull/432/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>